l10n_it_ricevute_bancarie/tests/test_riba.py

```diff
@@ -48,6 +48,17 @@ def test_not_add_due_cost(self):
         # ---- first due date for partner
         self.assertEqual(len(self.invoice2.invoice_line_ids), 1)
 
+    def test_not_add_due_cost_for_partner_exclude_expense(self):
+        # ---- Set Service in Company Config
+        self.invoice.company_id.due_cost_service_id = self.service_due_cost.id
```

Comment:
Assegni un id ad un record?

Reply:
se spacca il test, tolgo l'id.
Mi sono limitato a copiare l'assegnazione dagli altri test che precedono e seguono questo

Reply:
> Assegni un id ad un record?

Cosa ci vedi di strano? Sta scrivendo il campo `self.invoice.company_id.due_cost_service_id`

Reply:
Davo per scontato che ad un record si assegnasse un record, guardando un po' in giro però vedo che entrambe le assegnazioni sono usate (record o id). Come non detto.